Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -222,9 +222,9 @@ void _delayedEndFigureCall() {
     // closed.
     bool m_hasGeometryStarted;
     // True if this geometry can end its own figures.
-    bool m_allowsFigureCalls;
+    int m_allowsFigureCalls;
```

Comment:
@rajsesh-msft I think he's going from bool to "stack" -- to allow for nested calls to enable/disable figure state without trampling eachother